bpaotu/bpaotu/views.py

```diff
@@ -593,3 +613,115 @@ def dev_only_ckan_check_permissions(request):
     response = '||'.join([digest, data])
 
     return HttpResponse(response)
+
+
+# --------------------------------------------------------------------------------
+# Add Images to Map
+# --------------------------------------------------------------------------------
+
```

Comment:
I don't really like these block comments, I think if a bunch of stuff is specific to map images you should make a file for that, then it's obvious that the helper functions work within that context

Reply:
Moved all related code out into a different file.